.github/native-tests.json

```diff
@@ -83,7 +83,7 @@
         {
             "category": "Misc1",
             "timeout": 65,
-            "test-modules": "maven jackson jsonb jsch jgit quartz qute consul-config logging-min-level-unset logging-min-level-set"
+            "test-modules": "maven jackson jsonb jsch jgit quartz qute consul-config logging-min-level-unset logging-min-level-set simple___with___space"
```

Comment:
Maybe it's not a good idea but wouldn't it be better to introduce a comma separator rather than that approach? Would it be a problem for something else?

Reply:
Tagging @zakkak as I think he's consuming this too.

Reply:
I'll check the comma approach!

Reply:
Thanks for the tag @gsmet .

Do I understand correctly that you suggest using commas instead of spaces to separate the test modules, and thus avoid the need to use underscores in "simple with space"?

If so, I think that would be a nice improvement and I can probably do some check to handle it properly in my workflows (to allow for both formats till we no longer need to test 1.11.x). 

So it's OK for me.

Reply:
Using `, ` already in the json file makes things much easier, actually! Thanks @gsmet for this suggestion.
I'll push an update in the next few days.

Reply:
Commas are now present in the JSON file already.